web/src/components/customer/listing-details/contexts/CommitContext.tsx

```diff
@@ -139,12 +143,14 @@ const CommitContextProvider: React.FC<CommitContextProps> = ({
       return;
     }
 
+    onOpenPrompt('loading');
     try {
       await deleteCommit(commitId, idToken);
 
       setCommitId(undefined);
       await refetchCustomer();
       setCommitStatus(undefined);
+      onClosePrompt();
     } catch {
       onOpenPrompt('error');
```

Comment:
just to confirm, this reuses the loading prompt instead of creating a new error prompt on top of loading prompt?

Reply:
@ifhuang Yea it does not create another prompt on top of the previous one! It will just replace the content!